Announcing POSE proposal withdrawal

OscarSiba opened the pull request:
Added a subtitle to the call to action section at the end. Please review and see if you're with everything.

maneesha commented:
Thanks for thinking of this refactorization @fmichonneau 
Does this mean that `workshop_table.md` gets rebuilt with every page load?

fmichonneau commented:
the table gets built for each page where it's included. The template (`workshop_table.md`) requires one variable to be defined before calling `{% include %}` in the page. I think it's ready to merge if it looks good to you.

OscarSiba committed:
Annoucing POSE proposal

OscarSiba committed:
Update announcing-withdrawal-of-nsf-pose-proposal.md

Change title case, and quoted text indenting.

OscarSiba committed:
Update announcing-withdrawal-of-nsf-pose-proposal.md

further modified quoted text.

OscarSiba committed:
Update announcing-withdrawal-of-nsf-pose-proposal.md

added - to nonprofit.

ErinBecker commented:
Looks good to me!

OscarSiba commented:
Please merge @maneesha.

maneesha committed:
use relative urls and site variables

maneesha commented:
@OscarSiba There is a typo on the last bullet:
> Continue being an advocate for The Carpentries’ within your local community!

The apostrophe (`’`) after Carpentries signifies a possessive noun so either there is a missing word or the apostrophe should not be there.

Also the bulleted points end in periods (full stops) as per our style guide.  When that is also a hyperlink, sometimes the period is included in the hyperlinked text and sometimes not.  It doesn't break anything, but should we be consistent about that?  I think it looks cleaner when it is not, but defer to your judgement. 

Edit to add - I saw this is supposed to be published today but it only reached me on Friday.  I will merge once we resolve the typos as noted above.

OscarSiba committed:
Update announcing-withdrawal-of-nsf-pose-proposal.md

Made changes to the bulleted list at the end in light of @maneesha's last comment.

OscarSiba commented:
I've made these changes, all of them spot-on as usual, @maneesha, thank you.
